src/vmm/src/devices/virtio/queue.rs

```diff
@@ -106,8 +109,8 @@ impl<'a, M: GuestMemory> DescriptorChain<'a, M> {
             return None;
         }
 
-        let desc_head = mem.checked_offset(desc_table, (index as usize) * 16)?;
-        mem.checked_offset(desc_head, 16)?;
+        // There's no need for checking as we already validated the descriptor table and index bounds.
+        let desc_head = desc_table.unchecked_add((index as u64) * 16);
```

Comment:
We don't allow `as` conversions, so you need to replace `index as u64` -> `u64::from(index)`

Reply:
Thank you! The commit body is added as you suggested.

However, my `cargo fmt` didn't find the comment width issue. I believe it is because I didn't corrently setup git hook.

But to make it work for others like me, I'm wondering if `/tests/fmt.toml` can be moved or copied to `/rustfmt.toml`? It will be used automatically when `cargo fmt`. I can raise a PR if this acceptable.

Reply:
Sadly, specifying the option in rustfmt.toml would require a nightly compiler (at least last time I checked). However, we do have `./tools/devtool fmt`, which does use `fmt.toml` and should format everything for you :)